easybuild/framework/easyconfig/format/yeb.py

```diff
@@ -127,3 +127,13 @@ def is_yeb_format(filename, rawcontent):
             if line.startswith('name: '):
                 isyeb = True
     return isyeb
+
+# join functionality:
+# from http://stackoverflow.com/questions/5484016/how-can-i-do-string-concatenation-or-string-replacement-in-yaml/23212524#23212524
+# define custom tag handler
+def join(loader, node):
```

Comment:
`join` is maybe a bit too generic of a name for this function, maybe use `yaml_join`?